### ISSUE TITLE
Feature request: add in alternative references

### ISSUE THREAD

**Issue by swvanderlaan** [slowkow/proxysnps#13]
Hi,

It would be great to have (detailed) description of what is needed to get this package to work with alternative references. For instance, in-house references, or HRC, or UKBiobank, etc.

Thanks and best,

Sander

**Comment by slowkow** [comment]
I totally agree, and I think you might find this notebook useful regarding using other VCF files: https://github.com/slowkow/proxysnps/issues/12#issuecomment-450423002
